lms/djangoapps/oauth_dispatch/views.py

```diff
@@ -74,7 +74,10 @@ def _get_client_id(self, request):
         """
         Return the client_id from the provided request
         """
-        return request.POST.get('client_id')
+        client_id = request.POST.get('client_id')
+        if not client_id:
+            client_id = request.GET.get('client_id')
```

Comment:
We need to use URL parameters for GET requests, but I have updated to explicitly check the request method is GET before checking the URL parameters.